src/rules/function-no-interpolation/README.md

```diff
@@ -0,0 +1,64 @@
+# function-no-interpolation
+
+Since the release of [first-class `calc()`](https://sass-lang.com/documentation/values/calculations/),
```

Comment:
Please check the developer guide for REAME format. You can also have a look at the existing stylelint rules:

- https://stylelint.io/developer-guide/rules#write-the-readme
- https://github.com/stylelint/stylelint/tree/main/lib/rules

Reply:
Awesome, thanks for the resources, just fixed the README.md files in all PRs. 
Although.. I didn't find a rule similar to this rule (maybe [`function-calc-no-unspaced-operator`](https://github.com/stylelint/stylelint/tree/main/lib/rules/function-calc-no-unspaced-operator)?). Please let me know if I'm missing something. 
Thanks! :)

Reply:
@kristerkari Any thoughts on this ? :)

Reply:
Thanks for the fixes @pamelalozano16 ! 

One thing that I was still thinking about this `function-no-interpolation` rule: the name of this rule suggests that it could apply to all functions, but it only applies to the "calculation" functions, so maybe that should be part of the name to avoid possible confusion?

Reply:
The other PRs looked good now, so I can merge + cut a new release once this PR gets fixed.

Reply:
Maybe `function-calculation-no-interpolation` could a good name?

Reply:
Done! :)